packages/form-schema/src/components/CheckboxesWidget.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function selectValue(value, selected, all) {
+  const at = all.indexOf(value);
+  const updated = selected.slice(0, at).concat(value, selected.slice(at));
+  // As inserting values at predefined index positions doesn't work with empty
+  // arrays, we need to reorder the updated selection to match the initial order
+  return updated.sort((a, b) => all.indexOf(a) > all.indexOf(b));
+}
+
+function deselectValue(value, selected) {
+  return selected.filter(v => v !== value);
+}
+
+function CheckboxesWidget(props) {
+  const { id, disabled, options, value, autofocus, readonly, onChange } = props;
+  const { enumOptions, enumDisabled, inline } = options;
+  return (
+    <div className="checkboxes" id={id}>
+      {enumOptions.map((option, index) => {
+        const checked = value.indexOf(option.value) !== -1;
+        const itemDisabled = enumDisabled && enumDisabled.indexOf(option.value) !== -1;
+        const disabledCls = disabled || itemDisabled || readonly ? 'disabled' : '';
+        const checkbox = (
+          <span>
+            <input
```

Comment:
For this input, you will want to import the bcgov Checkbox component and use that instead, removing type="checkbox"